tests/scripts/test_dpanon.py

```diff
@@ -122,12 +122,12 @@ def test_dir_with_dt():
 
 def test_phoenix_root_bids_with_wrong_dt(phoenix_structure_BIDS):
     # with datatype
-    args = parse_args(['-p', 'tmp_phoenix',
-                       '-dt', 'haha',
-                       '-b'])
+    with pytest.raises(SystemExit):
```

Comment:
This line is necessary because after the proposed modification, the `dpanon.py` will fail at the beginning.